vcfconcat.c

```diff
@@ -549,6 +553,114 @@ static void concat(args_t *args)
     }
 }
 
+BGZF *hts_get_bgzfp(htsFile *fp);
+static void naive_concat(args_t *args)
+{
+    // only compressed BCF atm
+
+    BGZF *bgzf_out;
+    if ( strcmp("-",args->output_fname) )
+        bgzf_out = bgzf_open(args->output_fname,"w");
+    else
+        bgzf_out = bgzf_dopen(fileno(stdout), "w");
+
+    int page_size = getpagesize();
+    char *buf = (char*) valloc(page_size);
```

Comment:
I don't like solution hardwiring 32K as in https://github.com/samtools/htslib/commit/45c14cc37c65577a75d4043562f832cece38e13f. If portability is a concern, then the recommended replacement of `getpagesize` is `sysconf(_SC_PAGESIZE)` from `unistd.h`. Similarly `valloc` can be replaced with `posix_memalign`.